### PR TITLE
[UXE-4690] feat: integrate new stacked area chart to metrics

### PR DESCRIPTION
**Pull request by peterpaulo-azion**
### What is the new behavior introduced by this PR?
* stacked area chart (no more stacked line)

### Does this PR introduce breaking changes?
- [x] No
- [ ] Yes 

### Does this PR introduce UI changes? Add a video or screenshots here.
image

### Does it have a link on Figma?

<hr />

#### Make sure your pull request fits the checklist below (when applicable):

- [x] The issue title follows the format: [ISSUE_CODE] TYPE: TITLE
- [x] Commits are tagged with the right word (feat, test, refactor, etc)
- [x] Application responsiveness was tested to different screen sizes
- [x] Code is formatted and linted
- [x] Tags are added to the PR

#### These changes were tested on the following browsers:
- [x] Chrome
- [x] Edge
- [ ] Firefox
- [ ] Safari

### CODE REVIEW

File: src/modules/real-time-metrics/reports/convert-beholder-to-chart.js [hunk 1]
```diff
@@ -345,6 +370,64 @@ const formatMapChartData = ({ report, data }) => {
...
 
+const formatStackedAreaChart = ({ report, data }) => {
+  const dataset = Object.keys(data)
+
+  const timestamps = [
+    ...new Set(
+      data[dataset]
+        .map((entry) => {
+          const date = new Date(entry[report.xAxis])
+          return date.toISOString().split('T')[0] // Returns this date format: 'YYYY-MM-DD'
```
Comment (HerbertJulio) [inline src/modules/real-time-metrics/reports/convert-beholder-to-chart.js:422]: melhore o texto:
Sugiro remover o comentário e tornar o código mais claro:
```suggestion
          const entryDate = new Date(entry[report.xAxis]);
          const formattedDate = date.toLocaleDateString('en-CA')
         return formattedDate
```
OR
```suggestion
          const entryDate = new Date(entry[report.xAxis]);
          const formattedDate = entryDate.toISOString().split('T')[0];
          return formattedDate;
```

File: src/templates/graphs-card-block/components/chart/stacked-area-chart.vue [hunk 2]
```diff
@@ -7,23 +7,21 @@
...
-    }
+    chartData: Object,
+    resultChart: Array
   })
 
   const generate = () => {
     c3.generate({
-      bindto: `#stacked-line-chart-${props.data.id}`,
-      ...props.data
+      bindto: `#stacked-area-chart-${props.resultChart[0].id.id}`,
```
Comment (HerbertJulio) [inline src/templates/graphs-card-block/components/chart/stacked-area-chart.vue:16]: Realmente é `.id.id`?

File: src/templates/graphs-card-block/components/chart/stacked-area-chart.vue [hunk 3]
```diff
@@ -7,23 +7,21 @@
...
+      ...props.resultChart[0]
     })
   }
 </script>
 
 <template>
   <div
     class="[&>svg]:w-auto [&>svg]:flex"
-    :id="`stacked-line-chart-${props.data.id}`"
+    :id="`stacked-area-chart-${props.resultChart[0].id.id}`"
```
Comment (HerbertJulio) [inline src/templates/graphs-card-block/components/chart/stacked-area-chart.vue:25]: Realmente é `.id.id`?

Comment (peterpaulo-azion) [reply src/templates/graphs-card-block/components/chart/stacked-area-chart.vue]: Bem observado.

**Review by HerbertJulio — APPROVED** [review]

**Pull request merged**